puppetboard/templates/layout.html

```diff
@@ -16,6 +16,11 @@
     <link href='//fonts.googleapis.com/css?family=Open+Sans' rel='stylesheet' type='text/css'>
     {% endif %}
     <link href="{{ url_for('static', filename='Semantic-UI-2.1.8/semantic.min.css') }}" rel="stylesheet">
+    {% if config.OFFLINE_MODE %}
```

Comment:
I've made a small change here that has caused you a merge conflict.  

Can you combine the 2 {% if config.OFFLINE_MODE %} in the next patch?